core/src/main/java/nu/yona/server/util/TimeUtil.java

```diff
@@ -81,4 +81,9 @@ public static ZonedDateTime max(ZonedDateTime a, ZonedDateTime b)
 	{
 		return a.compareTo(b) > 0 ? a : b;
 	}
+
+	public static LocalDate max(LocalDate a, LocalDate b)
```

Comment:
We could add some unit tests on this simple method.

Reply:
Done